src/pages/Task.tsx

```diff
@@ -303,3 +311,12 @@ export const Task = withBannersContext(TaskCore);
 const LogWrapper = styled(PageLayout)`
   width: 100%;
 `;
+
+const getTabFromPath = (s: string): TaskTab | null => {
+  const parts = s.split("/");
+  const endPath = parts[parts.length - 1];
+  if (tabToIndexMap[endPath] !== undefined) {
+    return endPath as TaskTab;
+  }
+  return null;
+};
```

Comment:
You should use react-router to properly fetch the url param from the url.
https://reactrouter.com/web/api/Hooks/useparams

Reply:
done